### ISSUE TITLE
How to install more apps

### ISSUE THREAD

**Issue by mtran12** [leesah/docker-eclipse#2]
Hi leesah, I just pulled the image for eclipse and it's awesome. How can i install other app inside the image. I tried su and sudo but I dont know the passwd.

**Comment by leesah** [comment]
Hi @mtran12 thanks for encouraging. Honestly I didn't think there would be people using this image. Lol.
Well, as you can see in the Dockerfile, the user is created with `--disabled-password` argument, so it doesn't have a password. May I ask what software you want to install, and why do you want to use Eclipse in a container?

**Comment by mtran12** [comment]
I tried to install with apt-get but it either required sudo password for user, or su password. Blank password is not accepted so I'm stuck there. I just wanted to install a browser because every times I click on a link inside eclipse market or tutorial, it will give browser init failed coz there is no browser.

Eclipse in a container just perfect for me. I want to keep it isolated and portable, and same development environment for everyone. I tried other few images but yours gives the smoothest feeling.
